helix-query.yaml

```diff
@@ -22,3 +22,156 @@ indices:
         select: head > meta[property="og:url"]
         value: |
           match(attribute(el, 'content'), 'https:\/\/[^/]+(\/.*)')
+
+  seo-au:
+    <<: *seo
+    include:
+      - 'au/acrobat/**'
+      - 'au/creativecloud/**'
+      - 'au/sign/**'
+    exclude:
+      - 'de/**'
+      - 'en/**'
+      - 'es/**'
+      - 'fr/**'
+      - 'it/**'
+      - 'jp/**'
+      - 'kr/**'
+      - 'ru/**'
+    target: '/hub/query-index-au.xlsx'
+
+  seo-de:
+    <<: *seo
+    include:
+      - 'de/acrobat/**'
+      - 'de/creativecloud/**'
+      - 'de/sign/**'
+    exclude:
+      - 'au/**'
+      - 'en/**'
+      - 'es/**'
+      - 'fr/**'
+      - 'it/**'
+      - 'jp/**'
+      - 'kr/**'
+      - 'ru/**'
+    target: '/hub/query-index-de.xlsx'
+
+  seo-en:
+    <<: *seo
+    include:
+      - 'acrobat/**'
+      - 'creativecloud/**'
+      - 'sign/**'
+    exclude:
+      - 'au/**'
+      - 'de/**'
+      - 'es/**'
+      - 'fr/**'
+      - 'it/**'
+      - 'jp/**'
+      - 'kr/**'
+      - 'ru/**'
```

Comment:
and again, not required: none of them matches `acrobat/**`, `creativecloud/**` or `sign/**`